cfpb/capital-framework
# Update docs for Capital Framework v7.3.0 + other hygiene

Also includes some GitHub Pages hygiene updates.

## Additions

- Restores the copying of the icon font files into the static folder until we can totally kill it dead

## Removals

- Several HTML and markdown files that were no longer used

## Changes

- Updates usage files with the latest updates from Capital Framework
- Updates all icons that aren't part of the legacy cf-icons docs to be SVG
- Pulled out common top- and bottom-of-page stuff into `page-top.html` and `page-bottom.html` includes, so that that stuff isn't duplicated between the `default.html` and `home.html` layouts
- Replaced usage of `site.github.url` with the `relative_url` filter per [the latest guidance in Jekyll's docs](https://jekyllrb.com/docs/github-pages/#project-page-url-structure)
- Converts `{% highlight %}` tags to GFM-style fenced code blocks

## Testing

1. Pull branch
1. `npm run cf-unlink`, in case you have any local dev links still in place
1. `npm install`
1. `gulp`
1. `bundle update`
1. `bundle exec jekyll serve watch`
1. Load up http://localhost:4000/ and surf around; make sure nothing looks broken

## Notes

- Found one other extant use of the icon font in cf-core's usage.md file (the `.u-visually-hidden` example, but that needs to be taken care of upstream in the source file on canary/master.

## Checklist
